lib/generators/suspenders/jobs_generator.rb

```diff
@@ -0,0 +1,36 @@
+module Suspenders
+  module Generators
+    class JobsGenerator < Rails::Generators::Base
+      source_root File.expand_path("../../templates/active_job", __FILE__)
+      desc "Installs sidekiq for background job processing."
```

Comment:
[Minor] I think it's capitalized according to [their official website](https://sidekiq.org)

```suggestion
      desc "Installs Sidekiq for background job processing."
```

Reply:
Nice, thank you. I'm going to update all of them.